modules/repository-azure/src/main/java/module-info.java

```diff
@@ -18,18 +18,15 @@
     requires org.apache.logging.log4j;
     requires org.apache.logging.log4j.core;
 
-    requires com.azure.core;
     requires com.azure.http.netty;
-    requires com.azure.storage.blob;
-    requires com.azure.storage.common;
     requires com.azure.identity;
 
     requires io.netty.buffer;
     requires io.netty.transport;
     requires io.netty.resolver;
     requires io.netty.common;
 
-    requires reactor.core;
     requires reactor.netty.core;
     requires reactor.netty.http;
+    requires com.azure.storage.blob.batch;
```

Comment:
IntelliJ seemed to optimize the `requires`, the ones removed above are all transitively required by `com.azure.storage.blob.batch`.